digdag-docs/src/command_reference.rst

```diff
@@ -350,6 +350,8 @@ In the config file, following parameters are available
 * log-server.s3.credentials.secret-access-key (string. default: instance profile)
 * log-server.s3.path-style-access (boolean. default: false)
 * digdag.secret-encryption-key = (base64 encoded 128-bit AES encryption key)
+* executor.task_ttl (string. default: 1d. A task is killed if a single task is running longer than this period.)
```

Comment:
It might be better to use the same expression (`s/a single task/it/`) with the below one?

Reply:
Thank you for the reply. fixed it.